src/services/rhsm/rhsmSchemas.js

```diff
@@ -284,7 +284,7 @@ const tallyResponseSchema = Joi.object().keys({
 const rhsmSchemas = {
   capacity: response => schemaResponse({ response, schema: capacityResponseSchema, id: 'RHSM capacity' }),
   errors: response => schemaResponse({ response, schema: errorResponseSchema, id: 'RHSM errors' }),
-  guests: response => schemaResponse({ response, casing: 'camel', schema: guestsResponseSchema, id: 'RHSM guests' }),
+  guests: response => schemaResponse({ response, schema: guestsResponseSchema, id: 'RHSM guests' }),
```

Comment:
required:
confirm the instance id is within schema checks

Reply:
Confirmed, the instance_id is within the schema check for `instancesItem`, see https://github.com/RedHatInsights/curiosity-frontend/blob/main/src/services/rhsm/rhsmSchemas.js#L180